zstd-safe/zstd-sys/build.rs

```diff
@@ -159,9 +159,25 @@ fn compile_zstd() {
     #[cfg(feature = "thin")]
     {
         // Here we try to build a lib as thin/small as possible.
-        config.define("HUF_FORCE_DECOMPRESS_X1", Some("1"));
-        config.define("ZSTD_FORCE_DECOMPRESS_SEQUENCES_SHORT", Some("1"));
-        config.define("ZSTD_NO_INLINE ", Some("1"));
```

Comment:
This is the typo which #187 intended to fix.

However, with this PR we switch to using `ZSTD_LIB_MINIFY` so this is fixed anyway.